grails-app/conf/Config.groovy

```diff
@@ -1508,39 +1508,34 @@ openboxes {
-                        [label: "categories.label", defaultLabel: "Categories", href: "/${appName}/category/tree"],
-                        [label: "product.components.label", defaultLabel: "Components", href: "/${appName}/productComponent/list"],
-                        [label: "productGroups.label", defaultLabel: "Generic Products", href: "/${appName}/productGroup/list"],
-                        [label: "inventoryLevels.label", defaultLabel: "Inventory Levels", href: "/${appName}/inventoryLevel/list"],
-                        [label: "productType.label", defaultLabel: "Product Type", href: "/${appName}/productType/list", requiredRole: [RoleType.ROLE_SUPERUSER]]
+                        [label: "product.create.label", defaultLabel: "Create product", href: "/${appName}/product/create"],
+                        [label: "products.list.label", defaultLabel: "List Products", href: "/${appName}/product/list"],
+                        [label: "product.batchEdit.label", defaultLabel: "Batch edit product", href: "/${appName}/product/batchEdit"],
+                        [label: "product.importAsCsv.label", defaultLabel: "Import products", href: "/${appName}/product/importAsCsv"],
+                        [label: "product.exportAsCsv.label", defaultLabel: "Export products", href: "/${appName}/product/exportAsCsv"],
                     ]
                 ],
                 [
                     label: "", // No label
                     defaultLabel: "", // No label
                     menuItems: [
-                        [label: "products.label", defaultLabel: "Products", href: "/${appName}/product/list"],
-                        [label: "productSuppliers.label", defaultLabel: "Products Sources", href: "/${appName}/productSupplier/list"],
-                        [label: "product.associations.label", defaultLabel: "Products Associations", href: "/${appName}/productAssociation/list"],
+                        [label: "categories.label", defaultLabel: "Categories", href: "/${appName}/category/tree"],
+                        [label: "product.catalogs.label", defaultLabel: "Catalogs", href: "/${appName}/productCatalog/list"],
                         [label: "product.tags.label", defaultLabel: "Tags", href: "/${appName}/tag/list"],
-                        [label: "unitOfMeasure.label", defaultLabel: "Unit of Measure", href: "/${appName}/unitOfMeasure/list"],
-                        [label: "unitOfMeasureClass.label", defaultLabel: "Uom Class", href: "/${appName}/unitOfMeasureClass/list"],
-                        [label: "unitOfMeasureConversion.label", defaultLabel: "Uom Conversion", href: "/${appName}/unitOfMeasureConversion/list"]
+                        [label: "attributes.label", defaultLabel: "Attributes", href: "/${appName}/attribute/list"],
+                        [label: "product.associations.label", defaultLabel: "Products Associations", href: "/${appName}/productAssociation/list"],
```

Comment:
Should be only `Associations` in the label

Reply:
nice catch - in the messages file it was "Associations" anyway, so only default message was set as Products Associations before, so I didn't realize the default message is wrong.